engine-spring/core/src/main/java/org/camunda/bpm/engine/spring/SpringBeansResolverFactory.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.engine.spring;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.camunda.bpm.engine.delegate.VariableScope;
+import org.camunda.bpm.engine.impl.scripting.engine.Resolver;
+import org.camunda.bpm.engine.impl.scripting.engine.ResolverFactory;
+import org.springframework.context.ApplicationContext;
+
+/**
+ * <p>
+ * The {@link ResolverFactory} and {@link Resolver}class to access all the beans
+ * managed by the Spring container, to make them available in scripting.
+ * </p>
+ * 
+ * <p>
+ * {@see org.camunda.bpm.engine.spring.SpringProcessEngineConfiguration#initScripting()}
+ * <p>
+ *
+ * @author Laszlo Palossy
```

Comment:
With our new contribution guidelines, we don't accept code with `author` tags in the JavaDocs. Would you be willing to remove your author tags?

Reply:
Excuse me, it is not resolved yet!

Reply:
Documentation is yet to come!

Reply:
Hi Nikola!

The documentation was changed:
1. In the menu *'Spring Framework'* the menuitem *'Expression Resolving'* was renamed to *'Spring Bean Resolving'*.
2. The explanation of the new feature was added to the end of the *expressions.md* file.
3. In the section *'Using Spring Beans in Expressions'* the references were outdated, i updated them too. 
  (The name of the referenced example xml file and the code in the class `Printer` has changed meanwhile.)

The pull request to the documentation: https://github.com/camunda/camunda-docs-manual/pull/301/commits/588f68ea84634a84bd5bdda7cec7a8208d5a540b

Please check it if is ok so!

Cheers:
   László

Reply:
Hi László,

Just checked the docs and the javadocs. Looks great, thanks!